starter-code/src/viking.js

```diff
@@ -1,11 +1,101 @@
 // Soldier
-class Soldier {}
+class Soldier {
+	constructor(health, strength) {
+		this.health = health;
+		this.strength = strength;
+	}
+
+	attack() {
+		return this.strength;
+	}
+
+	receiveDamage(damage) {
+		this.health -= damage;
+	}
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+	constructor(name, health, strength) {
+		super(name, health, strength);
+
+		this.name = name;
+		this.health = health;
+		this.strength = strength;
+	}
+
```

Comment:
in this viking constructor you did something redundant. when you called for super(name, health, strngth) super is what is coming from the class that the current one is extending.
1st, Soldier doesnt have a name :)
and after that, you redeclared the values with this.health = health and this.strength = strength. By calling the super you dont need to redefine these values :)